tests/conftest.py

```diff
@@ -550,8 +550,10 @@ def search_service():
 
 
 @pytest.fixture
-def domain_status_service():
-    return account_services.NullDomainStatusService()
+def domain_status_service(mocker):
+    service = account_services.NullDomainStatusService()
+    mocker.spy(service, "get_domain_status")
```

Comment:
Note that the fixture sets up the tracking internally, and doesn't change what's returned by calling the fixture, only adds some methods.

If I try to spy on a method that doesn't exist, an `AttributeError` is raised, so that protects us from spying on things that don't exist, whereas a `call_recorder` on a created stub offers no such protection.